code_of_conduct.md

```diff
@@ -0,0 +1,5 @@
+## Kubernetes Community Code of Conduct
```

Comment:
I guess this doesn't make much difference. I see that other projects in k8s org also use the same sentence. We can keep it this way then.